views/job-roles.html

```diff
@@ -24,4 +24,5 @@
             </tr>
         {% endfor %}
     </table>
+    <a href="/create-job-role" class="white btn btn-primary">Create</a>
```

Comment:
Can you add a little bit of space between this button and the table? Also if the table is really long then this button won't be on the screen so maybe positioning it at the top would be better